sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>We need a short intro paragraph that includes a motivation and purpose of this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Agreed, and I think the notebook title can be clarified as well. Suggestion as a starting point: <code>Find the maximum batch size for fastest training time with SageMaker Training Compiler</code>  </p>